ethpm_types/source.py

```diff
@@ -1,10 +1,22 @@
-from typing import Iterator, List, Optional, Union
```

Comment:
most additions are in this file

Reply:
altho nothing really different if you already reviewed the core PR, just moved some stuff in here and may have improved some of it as i was adding some more tests

Reply:
Yeah, it's probably easier to test more freely here without bloating the core test suite